com.unity.render-pipelines.universal/Editor/Converter/RenderPipelineConverter.cs

```diff
@@ -2,6 +2,7 @@
 using System.Runtime.CompilerServices;
 
 [assembly: InternalsVisibleTo("PPv2URPConverters")]
+[assembly: InternalsVisibleTo("Unity.2D.PixelPerfect.Editor")]
```

Comment:
Which internal API do you need access?

Reply:
The RenderPipelineConverter class. Inheriting from it so it will be a part of the Upgrade 2D (URP) Assets in the Render Pipeline Converter. See https://github.cds.internal.unity3d.com/unity/2d/pull/1514 (not sure if you have access)